server/pxf-api/src/main/java/org/greenplum/pxf/api/configuration/PxfJksTextEncryptorConfiguration.java

```
@@ -0,0 +1,30 @@
+package org.greenplum.pxf.api.configuration;
+
+import io.arenadata.security.encryption.client.configuration.JksTextEncryptorConfiguration;
+import org.springframework.beans.factory.annotation.Value;
+
```

```java
@Configuration
@ConditionalOnProperty({"pxf.ssl.jks-store.path", "pxf.ssl.jks-store.password", "pxf.ssl.salt.key"})
```

---

Added annotation to this class. 

But it seems that if we have wrong parameter values or there is some error to get encryption key from the store we will receive error to load bean and PXF service will not start. Is it expected behavior?

`org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'textEncryptorProvider' defined in class path resource [org/greenplum/pxf/api/configuration/PxfJksTextEncryptorConfiguration.class]`

---

Yes, of course! We should throw an exception as early as possible if user has entered invalid settings.